src/main/java/uk/gov/hmcts/reform/adoption/controller/StartCaseController.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.adoption.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.adoption.service.CoreCaseDataService;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+@RestController
+@RequestMapping("/")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class StartCaseController {
+    private final CoreCaseDataService coreCaseDataService;
+
+    @PostMapping("case")
+    public CaseDetails startCase() {
+        return coreCaseDataService.startCase();
```

Comment:
ideally we should prevent creation of multiple cases by one user

Reply:
yes, so I want to get this merged first. The next step is then an if case exists for user. There is some difficulty in this I have been discussing with Adam. To keep you in the loop, there is a v1 core case data api endpoint which will get cases for a user, but is for caseworkers - so will require another user to access. It also does not exist in ccd-client we are using. Other options involve elastic search and either storing creatorId in case data or asking ccd to add creatorId to case metadata